src/main/assembly/dist/bag/metadata/amd/v1.0/wfs.xsd

```diff
@@ -0,0 +1,175 @@
+                <xs:element name="completionTime" form="unqualified" type="xs:dateTime">
+                    <xs:annotation>
+                        <xs:documentation>The timestamp of marking this flow-step as completed.</xs:documentation>
+                    </xs:annotation>
+                </xs:element>
+                <xs:element name="doneById" form="unqualified" type="xs:NCName">
+                    <xs:annotation>
+                        <xs:documentation>The DANS data manager responsible for this flow-step</xs:documentation>
+                    </xs:annotation>
+                </xs:element>
+                <xs:element name="id" form="unqualified" type="wfs:WorkflowStepType"/>
+                <xs:element name="remarks" form="unqualified">
+                    <xs:complexType>
+                        <xs:sequence>
+                            <xs:element minOccurs="0" name="remark" form="unqualified">
+                                <xs:complexType>
+                                    <xs:sequence>
+                                        <xs:element name="text" form="unqualified" type="xs:string"/>
+                                        <xs:element name="remarkerId" form="unqualified" type="xs:NCName"/>
+                                        <xs:element name="remarkDate" form="unqualified" type="xs:dateTime"/>
+                                    </xs:sequence>
+                                </xs:complexType>
+                            </xs:element>
+                        </xs:sequence>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="required" form="unqualified" type="xs:boolean"/>
+                <xs:element name="steps" form="unqualified">
+                    <xs:complexType>
+                        <xs:sequence>
+                            <xs:element minOccurs="0" maxOccurs="unbounded" ref="wfs:workflow"/>
+                        </xs:sequence>
+                    </xs:complexType>
+                </xs:element>
+            </xs:choice>
+        </xs:complexType>
+    </xs:element>
+    <xs:simpleType name="WorkflowStepType">
+        <xs:restriction base="xs:NMTOKEN">
+            <xs:enumeration value="dataset">
+                <xs:annotation>
+                    <xs:documentation/>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="dataset.sip">
+                <xs:annotation>
+                    <xs:documentation/>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="dataset.sip.files">
+                <xs:annotation>
+                    <xs:documentation/>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="dataset.sip.files.completeness">
+                <xs:annotation>
+                    <xs:documentation>Controle op compleetheid aanlevering</xs:documentation>
```

Comment:
why is this and following documentations in Dutch? Change to English, please.

Reply:
this is the exact text that the datamanagers agree to have checked in the webui. 
i'd rather keep that, could add an english translation?

Reply:
👍 add the English translation